crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -255,6 +437,63 @@ def proposals
   api :GET, "/crowbar/:barclamp/1.0/proposals/template",
     "Returns the content of a proposal template"
   param :barclamp, String, desc: "Name of the barclamp", required: true
+  example '
+  {
+    "id": "template-dns",
```

Comment:
You can create tech debts to track that work separately and even in a different sprint, as long as PO (Vincent) is ok with it. I'm only raising issues as I see them.